docs/01-about/index.md

```diff
@@ -1,37 +1,58 @@
+# What is Nordcraft?
 
-## Bridging design and development
-Nordcraft combines a visual editor for interface design with a powerful technical foundation built on modern web standards. This approach accelerates development without sacrificing quality or control.
+Nordcraft is an open-source **Web Development Engine** that combines visual design and full programming capabilities in a single platform. Nordcraft eliminates the traditional handoff between designers and developers, creating a seamless workflow where everyone can collaborate effectively and efficiently **without limits**.
 
-Key capabilities include:
-- Visual editing of UI components and application logic
+::: info
+Nordcraft is for designers **and** developers, whether you're working on your next big side project, or as part of a professional product team.
+:::
+
+## Unifying design and development
+
+Nordcraft combines a visual editor for interface design with a powerful technical foundation built on modern web standards, accelerating development of your websites and web applications without sacrificing quality or control.
+
+Key features include:
+
+- Visual editing of UI components familiar to designers working in Figma
 - Component-based architecture familiar to React, Vue, and Svelte developers
-- Visual development for implementing interactive behavior
-- Branch-based development for team collaboration
+- Visual development environment for implementing interactive behavior and application logic
+- Branch-based environment for parallel feature development and team collaboration
 
 ## Technical foundation
-Built on a lightweight 24kb runtime with a reactive architecture, Nordcraft supports both server-side and client-side rendering. The system prioritizes performance while providing developers with the tools they need for modern web applications.
+
+The Nordcraft Web Development Engine is a lightweight 24kb runtime with a reactive architecture, supporting both server-side and client-side rendering. Nordcraft prioritizes the performance of your applications while providing you with all the tools you need for modern web application development.
 
 Core technical features:
+
 - Signals-based reactive programming model
 - Full server-side rendering capabilities
 - Minimal DOM output for optimal performance
 - Animation engine for creating dynamic interfaces
 - Integrated version control with preview environments
+- Add your own custom code to your Nordcraft apps
 
-## Complete ecosystem
-Nordcraft provides everything needed to build and deploy web applications:
-- API integration for connecting to any backend service
+## The Nordcraft ecosystem
+
+Nordcraft provides everything you need to build, test, and deploy modern, performant web applications:
```

Comment:
Not sure if we should highlight test here?

Reply:
I figured you'd say that 😄